modules/virt-understanding-smart-cloning.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-cloning-a-datavolume-using-smart-cloning.adoc
+
+[id="virt-understanding-smart-cloning_{context}"]
+= Understanding smart cloning
+
+When a DataVolume (DV) is smart-cloned, the following occurs:
+
+. A snapshot of the source PersistentVolumeClaim (PVC) is created.
+. A PVC is created from the snapshot.
+. The snapshot is deleted.
+
+== Prerequisites
```

Comment:
Having a heading of "Prerequisites" in a module made me think that this was a procedure module, and that this should have been ".Prerequisites". 

However, if this is intended to more informational than actionable, then I would try rewording to get a bit more detailed, explaining that they are the prerequisites for smart cloning.

Reply:
Just checking.... I thought that .Prerequisites was being deprecated in favor of == Prerequisites. But I see what you mean. :-)

Changed to (assuming I can make "requirements..." a heading 2):
== Requirements for smart-cloning

For smart-cloning to occur, the following must be true.

* Your storage provider supports snapshots.
* The source and target PVCs are defined to the same namespace.
* The source and target PVCs are defined to the same storage class.
* The VolumeSnapshotClass object references the storage class defined to both the source and target PVCs.

If any of these requirements are not true, host-assisted cloning occurs when you create a DV with a
PVC source.

Reply:
No actually the only change to this came at the assembly level. We use == for Prerequisites and Additional resources in assemblies. But at the module level it should still be .Prerequisites.

We have the new guidance documented in our contrib guidelines here: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#writing-procedures

Reply:
AH! Makes sense. Thanks for clarifying that.